tests/testthat/test-sce_to_seurat.R

```diff
@@ -34,7 +36,32 @@ test_that("Converting SCE to Seurat objects works as expected", {
 
   # can't directly check that coldata is equal because of added columns in seurat object
   expect_true(all(colnames(coldata_sce) %in% colnames(seurat_object@meta.data)))
-  expect_equal(rowdata_sce, seurat_object[["RNA"]]@var.features)
+  expect_equal(rowdata_sce, seurat_object[["RNA"]]@var.features) # since default "counts" is used, RNA name is expected here
+  expect_equal(metadata(sce), seurat_object@misc)
+
+  # check that altExp data was converted and rowData
+  expect_s4_class(seurat_object[[alt_name]], 'Assay')
+  alt_rowdata_sce <- as.data.frame(rowData(altExp(sce, alt_name)))
+  expect_equal(alt_rowdata_sce, seurat_object[[alt_name]]@var.features)
+
+})
+
+
+test_that("Converting SCE to Seurat objects works as expected for custom assay name", {
+
+  seurat_object <- sce_to_seurat(sce, assay_name = "logcounts")
+
```

Comment:
I think I would also include an explicit check for the assay name being logcounts here? I don't know that you need the rest of the checks but they can't hurt to have since they should all pass if it's converted successfully.

Reply:
Done in 6dd5790!